### DISCUSSION TITLE
Search with regex

### DISCUSSION THREAD

**Discussion by andershagman** [netbox-community/netbox#8407] [Q&A]
Hi

Is there a chans to implement regex search in the gui.
I often find myself unable to filter the exact list I want.

Typical simple example:
esx181[5-9].*mgmt (interface list to only get switch interfaces connected to mgmt ports)

**Comment by jeremystretch** [comment]
FR #7604 has been opened to make [filter lookup expressions](https://netbox.readthedocs.io/en/stable/rest-api/filtering/#lookup-expressions) available in the UI, but I don't believe we support a regex lookup currently. That would need a separate FR.
